src/github-helper.ts

```diff
@@ -64,10 +64,10 @@ export async function createPullRequest(
 
     // Apply labels
     if (inputs.labels.length > 0) {
-      const prLabels: Label[] =
+      const prLabels =
```

Comment:
```suggestion
      const prLabels = github.context.payload?.pull_request?.labels ?? []
```

Null coalescence and optional chaining work well together.